granary/microformats2.py

```diff
@@ -356,23 +374,30 @@ def json_to_html(obj):
   prop = first_props(props)
   prop.setdefault('uid', '')
   author = prop.get('author')
-  if author:
-    author['type'].append('p-author')
 
   # if this post is itself a like or repost, link to its target(s). (do this and
   # rsvp below *before* content since they set props['name'] if necessary.)
   likes_and_reposts = []
-  for verb in 'like', 'repost':
-    if ('h-as-%s' % verb) in obj['type']:
-      if not props.get('name'):
-        props['name'] = ['%ss this.' % verb]
-      likes_and_reposts += ['<a class="u-%s u-%s-of" href="%s"></a>' %
-                            (verb, verb, url) for url in props.get(verb)]
+  if 'h-as-like' in obj['type']:
+    if not props.get('name'):
+      props['name'] = ['likes this.']
+    likes_and_reposts += ['<a class="u-like u-like-of" href="%s"></a>' %
```

Comment:
if there are multiple `like-of`s, then the values will be composite objects instead of string urls, due to line 190, right?